src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1384,8 +1384,37 @@ private String getContainerName() {
         return DEVMODE_CONTAINER_BASE_NAME + ((highestNum != -1) ? "-" + ++highestNum : "");
     }
 
+    private String[] getContainerNetworks(String contName) {
+        String dockerNetworkCmd = "docker inspect -f '{{.NetworkSettings.Networks}}' " + contName;
+        String cmdResult = execDockerCmd(dockerNetworkCmd, 10, false);
+        if (cmdResult == null || cmdResult.contains(" RC=")) { // RC is added in execDockerCmd if there is an error
+            warn("Unable to retrieve container networks.");
+            return null;
+        }
+        else {
+            // Example cmdResult value: map[bridge:0xc000622000 myNet:0xc0006220c0 otherNet:0xc000622180]
+            String networkMap = cmdResult.substring(cmdResult.indexOf("[") + 1, cmdResult.indexOf("]") -1);
+            String[] networkHex = networkMap.split(" ");
+            String[] networks = new String[networkHex.length];
+            for (int i=0; i < networkHex.length; i++) {
+                networks[i] = networkHex[i].split(":")[0];
+            }
+            return networks;
```

Comment:
It would be good to extract this parsing code into a separate method and add unit tests for it.